Code/GraphMol/MolDraw2D/DrawMol.cpp

```diff
@@ -1660,11 +1660,20 @@ void DrawMol::makeDoubleBondLines(
     // in, for example, an aldehyde, such as in catch_tests.cpp's
     // testGithub_5269_2.svg, might be asymmetrically shorter, so we don't
     // want to change colour at halfway
-    if (bond->getEndAtom()->getDegree() == 1 && !(cols.first == cols.second) &&
-        fabs((l1s - l1f).lengthSq() - (l2s - l2f).lengthSq()) > 0.01) {
-      double midlen = (l1s - l1f).length() / 2.0;
-      Point2D lineDir = l2s.directionVector(l2f);
-      Point2D notMid = l2s + lineDir * midlen;
+    auto l1 = (l1s - l1f).lengthSq();
+    auto l2 = (l2s - l2f).lengthSq();
+    if ((bond->getBeginAtom()->getDegree() == 1 ||
+         bond->getEndAtom()->getDegree() == 1) &&
+        !(cols.first == cols.second) && fabs(l1 - l2) > 0.01) {
```

Comment:
```suggestion
        (cols.first != cols.second) && fabs(l1 - l2) > 0.01) {
```

Reply:
Operator != isn't defined for DrawColour.  I have thought about making one, but it feels like a bit of code bloat to solve something that is a only a question of ugly style, really, and not used often.

Reply:
I usually define the `!=` operator negating the `==` operator as you did above. I think that makes code using the class a bit more readable as the ugliness stays in the operator definition, but it is definitely not a must.

Reply:
I will put one in.  I was in two minds about it when I wrote the code, and your extra vote has convinced me.